src/isar/config/settings.py

```diff
@@ -183,8 +183,11 @@ class Settings(BaseSettings):
     # Type of robot ISAR is monitoring
     ROBOT_TYPE: str = Field(default="robot")
 
-    # Name or unique ID of robot
-    ROBOT_ID: str = Field(default="R2-D2")
+    # Name of robot
+    ROBOT_NAME: str = Field(default="R2-D2")
+
+    # Unique identifier for this ISAR instance. Note that this should be a generated UUID.
+    ISAR_ID: str = Field(default="c68b679d-308b-460f-9fe0-87eaadbd8a6e")
```

Comment:
Maybe change the default to something like:
"00000000-0000-0000-0000-000000000000" to make it clear it is not generated automatically

Reply:
Done